lib/ansible/modules/dnf5.py

```diff
@@ -423,7 +423,9 @@ def is_newer_version_installed(base, spec):
 
     try:
         spec_nevra = next(iter(libdnf5.rpm.Nevra.parse(spec)))
-    except (LIBDNF5_ERROR, StopIteration):
+    except LIBDNF5_ERRORS:
```

Comment:
Why separate the exception (just curious to know)?

Reply:
I do not, just giving you an example in case you wanted it.